themes/brainy/README.md

```diff
@@ -1,8 +1,35 @@
 # Brainy theme
 
 Simple colorful terminal prompt theme (inspired by a number of themes).
+Supported on all operating systems
+In constant maintenance and improvement
 
-![alt text](https://www.lfsystems.xyz/img/brainy-Arch.png)
```

Comment:
Hi nwinkler,
ok, I have removed the image, I apologize, you are absolutely right the idea is that the framework is as stable as possible.